src/rpcgovernance.cpp

```diff
@@ -0,0 +1,637 @@
+// Copyright (c) 2014-2016 The Dash Core developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+//#define ENABLE_DASH_DEBUG
+
+#include "util.h"
+#include "main.h"
+#include "db.h"
+#include "init.h"
+#include "activemasternode.h"
+#include "darksend.h"
+#include "governance.h"
+#include "darksend.h"
+#include "masternode-payments.h"
+#include "masternode-sync.h"
+#include "masternodeconfig.h"
+#include "masternodeman.h"
+#include "rpcserver.h"
+#include "utilmoneystr.h"
+#include <boost/lexical_cast.hpp>
+
+#include <fstream>
+#include <iostream>
+#include <sstream>
+
+using namespace std;
+
+int ConvertVoteOutcome(std::string strVoteOutcome)
```

Comment:
`ConvertVoteOutcome` and `ConvertVoteSignal` look like a complete copy/paste of the same functions from `CGovernanceVoting`, I'm sure there is a better way ;)